tests/system/robot/chromeTests.py

```diff
@@ -220,7 +231,7 @@ def test_mark_aria_details_role():
 		actualBraille,
 		# TODO: fix known bug with braille not announcing details
 		# expectedBraille,

```

Comment:
What is the plan to deal with this?

I note it is listed in known issues, can you make sure that a bug is raised/linked for it please.

Reply:
I am yet to write up an issue for this.

Reply:
Issue created: #13815